app/components/vistk-scatterplot.js

```diff
@@ -179,15 +179,27 @@ export default Ember.Component.extend({
     if(this.get('dataType') === 'industries') { return 'employment'; }
   }),
   didInsertElement: function() {
-    $.getJSON(`${apiURL}/data/location?level=department`).then((response) => {
```

Comment:
Sorry about this,  It was a fast hack to get it done and I was going to go back and refactor.  But you know...